app/[locale]/blog/[slug]/__tests__/page.test.tsx

```
@@ -0,0 +1,842 @@
+        return () => {
+          if (rafId) {
+            cancelAnimationFrame(rafId)
+          }
+        }
+      }
+    }
+  }, [setNotificationWithTimeout])
+
+  return (
+    <div>
+      <button data-testid="share-button" onClick={handleShare}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)